tools/firmware.toit

```diff
@@ -906,6 +907,48 @@ extract_binary_content -> ByteArray
   binary.patch_extend_drom system_uuid table_address extension
   return binary.bits
 
+describe_cmd -> cli.Command:
+  return cli.Command "describe"
+      --short_help="Describe the contents of the given firmware envelope."
+      --options=[
+        cli.OptionEnum "output-format" ["text", "json"]
+            --default="text",
+        cli.Flag "all"
+            --short_help="Show all information, including system entries."
+            --short_name="a",
+      ]
+      --run=:: describe it
+
+describe parsed/cli.Parsed -> none:
+  input_path := parsed[OPTION_ENVELOPE]
+  output_format := parsed["output-format"]
+  show_all := parsed["all"]
+
+  envelope := Envelope.load input_path
+
+  if output_format == "text":
+    print "Envelope format version: $envelope.version_"
+    print "SDK version: $envelope.sdk_version"
+    print
+    print "Containers:"
+    envelope.entries.do: | name entry |
+      if show_all or not name.starts_with "\$":
+        print "  $name: $(entry.size) bytes"
```

Comment:
Don't we have more to show if the entry is a snapshot? Can you share this with code with 'container list'?